app/presenters/open_studios_event_presenter.rb

```diff
@@ -111,11 +111,11 @@ def special_event_time_range
                                 model.special_event_end_time)
   end
 
-  def for_display(reverse: false)
```

Comment:
Feels like some overlap between service and presenter?
maybe the service should return a presenter and that'd collapse some of this stuff